src/qt/pivx/masternodeswidget.cpp

```diff
@@ -133,7 +133,8 @@ MasterNodesWidget::MasterNodesWidget(PIVXGUI *parent) :
     connect(ui->btnAboutController, &OptionButton::clicked, [this](){window->openFAQ(10);});
 }
 
-void MasterNodesWidget::showEvent(QShowEvent *event){
+void MasterNodesWidget::showEvent(QShowEvent *event)
```

Comment:
Could you extract out these styling changes to their own commit?

Reply:
sure, can do.

Reply:
done